Documentation/QtCreatorConfiguration.md

```diff
@@ -11,8 +11,8 @@ First, make sure you have a working toolchain and can build and run SerenityOS.
 * Wait for the file list to generate. This can take a minute or two!
 * Ignore the file list, we will overwrite it later. Click Next.
 * Set `Add to version control` to `<None>`. Click Finish.
-* In your shell, go to your SerenityOS project directory, and invoke the `Meta/refresh-serenity-qtcreator.sh` script to regenerate the `serenity.files` file. You will also have to do this every time you delete or add a new file to the project.
```

Comment:
"SerenityOS project directory" needs changing too

Reply:
Can I do in a different PR?

Reply:
I guess you'll have to now. :sweat_smile: Andrew was too fast for me

Reply:
whoops :sweat_smile:

Reply:
No problem, i'll throw a new one :)